rpa-builder/6.1/modules/ROOT/pages/adding-a-pattern-from-screen-capture.adoc

```diff
@@ -0,0 +1,159 @@
+* *Define Search Area* 
++
+Enables you to reduce the search area from full screen to a smaller area by clicking and dragging the mouse. The selection is transparent red with a red border. When defining the search area, you can unfreeze and refreeze the screen.
+* *Reuse last defined Search Area* 
++
+Enables you to reuse a search area defined in a previous capture. This option shows only if you already defined a search area when setting up the Workflow. 
+* *Accept selection & continue* 
++
+Adds the selection into the wizard for the corresponding Action Step and then returns to that Action Step.
+
+After you capture at least one search pattern, the *Delete selected pattern* option shows. This option removes the border around the previously selected pattern (or the last search area defined).
+
+=== Unfreeze Screen 
+
+When you click *Unfreeze screen* the Capture Patterns window changes as follows:
+
+image::take-a-screenshot-window.png[30%, 30%, The Take a screenshot window that shows after clicking Unfreeze Screen.]
+
+* *Capture screen* 
++
+Freezes the screen image again so that the pattern can be captured. 
+* *Capture screen with delay* 
++
+Starts the timer. After a delay, the screen image refreezes and the Capture Patterns window reappears.
+
+[[using-color-spot-with-search-patterns]]
+== Using Color Spot with Search Patterns
+
+By default, RPA Builder only captures the pattern features for search patterns and color plays no role in pattern-based searches. However, sometimes you may find that it is necessary to determine when the color in a certain position in the search pattern is correct. For example, the text of an error message needs to be in red, or a traffic light symbol needs to appear green rather than yellow or red. In order to guarantee this, there is an option to define a Color Spot for a Search Pattern. In the case of a Search Pattern with Color Spot, the pattern-based image search is successful only if the color in a defined position matches a predefined color.
+
+When you hover a search pattern in the wizard for the relevant Action Step, an overlay menu bar shows at the end of the image. 
+
+image:search-pattern-icons.png[20%, 20%, A search pattern with the overlay menu that shows the available actions.]
+
+Click the *Color Spot Picker* (image:color-picker-icon.png[2%, 2%, The Color Spot Picker button.]) to open a window where you can select the color spot to use with the search pattern.
+
+image:color-spot-picker-window.png[60%, 60%, The Color Spot Picker window.]
+
+As you move the mouse over the image, the *Live values* section updates showing the following information: 
+
+* *Color*
++
+The color value, expressed in RGB color values (`R:255`, `G:255`, `B:255`), for the current image coordinates. 
+* *Coordinates*
++
+The image coordinates based on the current position of the mouse, indicated in `x;y` coordinates being `0;0` the upper-left corner of the image.
```

Comment:
Is there a way you can avoid positional words like "upper-left" for accessibility reasons? (I'm not sure if you can do that here.)